src/index.js

```diff
@@ -0,0 +1,92 @@
+  import React from 'react';
+  import ReactDOM from 'react-dom';
+
+  //start stitch import
+  import { Stitch, GoogleRedirectCredential } from "mongodb-stitch-browser-sdk";
+  //end stitch import
+  
+  //display google auth user information
+  class User extends React.Component {
+
+    constructor(props) {
+      super(props);
+    }
+
+    render() {
+      if (this.props.value === undefined) return null;
+      
+      return <div>
+              <pre>{this.props.value.data.name}</pre>
+              </div>
+    }
+  }
+
+  class Demo extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = { 
+        user: [],
+      }
+    }
+
+    componentDidMount() {
+      this.setupStitch();
+    }
+    
+    //start stitch setup
+    setupStitch() {
+      //copy the name of your google-auth enabled stitch application here
+      //the name of the app will typically be the stitch application name
+      //with a "-"" + random string appended
+      const appName = 'authentication_test-htbrq';
+
+      //need a description of how this works as the API method names are a
+      //little unclear
+      const stitchClient = Stitch.hasAppClient(appName) ? Stitch.defaultAppClient : Stitch.initializeDefaultAppClient(appName);
+      
+      //check if this user has already authenticated and we're
+      //here from the redirect. If so, process the redirect.
+      if (stitchClient.auth.hasRedirectResult()) {
+          stitchClient.auth.handleRedirectResult().then(user => {
```

Comment:
Not actually sure if this will set `client.auth.isLoggedIn = true` in time to work with the check on line 56. My intuition is that the check would need to happen inside the `.then()` here. Alternatively we can `await client.auth.handleRedirectResult()`.